tasks/build-chi-scripts.js

```diff
@@ -69,6 +69,7 @@ function buildChiScripts () {
     .pipe(plumber())
     .pipe(vinylNamed())
     .pipe(webpackStream(webpackConfig))
+    .pipe(footer("document.addEventListener('mousedown', function (e) { if (!document.body.classList.contains('-mouse-control')) { document.body.classList.add('-mouse-control'); } }); document.addEventListener('keyup', function (e) { if (e.keyCode === 9 && document.body.classList.contains('-mouse-control')) { document.body.classList.remove('-mouse-control'); } });"))
```

Comment:
You can use a keycode const here also

Reply:
Yep, good catch, I was waiting for the other PR to be merged. It is merged now, I'm on it 😉

Reply:
Oops, I just realized that we can not use the keys constant here since `.pipe(footer(""))` basically injects this functionality to the file from a string. Any ideas on taking another approach?

Reply:
Not really since I haven't worked with webpack pipes before, sorry 🤷🏻‍♂️

Reply:
OK, I believe this kind of string interpolation is the way to code:

import { KEYS } from '../src/chi/javascript/constants/constants';

...

.pipe(footer(`....... if (e.keyCode === ${KEYS.TAB} .......`))

What do you think?

Reply:
Nice! That deserves an ACK if works ;)